packages/pwa-kit-create-app/assets/bootstrap/js/README.MD

```diff
@@ -0,0 +1,54 @@
+<h1>PWA Kit Generated App</h1>
+
+Welcome to the PWA Kit!
+
+## Getting Started
+
+### Requirements
+
+-   Node 18 or later
+-   npm 9 or later
+
+### Run the Project Locally
+
+```bash
+npm start
+```
+
+This will open a browser and your storefront will be running on http://localhost:3000
+
+### Deploy to Managed Runtime
+
+```
+npm run push -- -m "Message to help you recognize this bundle"
+```
+
+**Important**: Access to the [Runtime Admin](https://runtime.commercecloud.com/) application is required to deploy bundles. To learn more, read our guide to [Push and Deploy Bundles](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/pushing-and-deploying-bundles.html).
+
+## Customizing the application
+
+This version of the application uses [Template Extensibility](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/template-extensibility.html) to empower you to more easily customize base templates. Please refer to our documentation for more information.
```

Comment:
This readme is only included for projects that use template extensibility. So I added a section about template extensibility here.

Reply:
Do this file apply for the project that users choose to not use extensibility?